[2025-04-11T12:52:47.915+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scholamigo_dag.run_erasmus_collector manual__2025-04-11T12:52:47.164535+00:00 [queued]>
[2025-04-11T12:52:47.918+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scholamigo_dag.run_erasmus_collector manual__2025-04-11T12:52:47.164535+00:00 [queued]>
[2025-04-11T12:52:47.918+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-11T12:52:47.922+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_erasmus_collector> on 2025-04-11 12:52:47.164535+00:00
[2025-04-11T12:52:47.924+0000] {standard_task_runner.py:60} INFO - Started process 201 to run task
[2025-04-11T12:52:47.926+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'scholamigo_dag', 'run_erasmus_collector', 'manual__2025-04-11T12:52:47.164535+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/scholamigo.py', '--cfg-path', '/tmp/tmpp8pcw2fy']
[2025-04-11T12:52:47.927+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask run_erasmus_collector
[2025-04-11T12:52:47.946+0000] {task_command.py:423} INFO - Running <TaskInstance: scholamigo_dag.run_erasmus_collector manual__2025-04-11T12:52:47.164535+00:00 [running]> on host 426a21ea82cf
[2025-04-11T12:52:47.970+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scholamigo_dag' AIRFLOW_CTX_TASK_ID='run_erasmus_collector' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T12:52:47.164535+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T12:52:47.164535+00:00'
[2025-04-11T12:52:47.971+0000] {logging_mixin.py:188} INFO - ðŸš€ Running: /opt/***/landing_zone/collectors/erasmus_collector.py
[2025-04-11T12:54:58.767+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-04-11T12:54:58.770+0000] {process_utils.py:131} INFO - Sending 15 to group 201. PIDs of all processes in the group: [202, 203, 201]
[2025-04-11T12:54:58.771+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 201
[2025-04-11T12:54:58.771+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-04-11T12:54:58.792+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=201, status='terminated', exitcode=0, started='12:52:47') (201) terminated with exit code 0
[2025-04-11T12:54:58.793+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=202, status='terminated', started='12:52:47') (202) terminated with exit code None
[2025-04-11T12:54:58.793+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=203, status='terminated', started='12:52:47') (203) terminated with exit code None
